domains/infrastructure/dns.tf

```diff
@@ -0,0 +1,5 @@
+module "dns" {
+  source      = "git::https://github.com/DFE-Digital/terraform-modules.git//dns/zones?ref=617-make-dns-set-up-more-generic"
```

Comment:
should this be pointing to a branch now @saliceti?

Reply:
Thanks!